code/game/turfs/turf.dm

```diff
@@ -320,3 +332,21 @@
 	SSair.remove_from_active(T0)
 	T0.CalculateAdjacentTurfs()
 	SSair.add_to_active(T0,1)
+
+
+/turf/proc/update_cliff()
+	if(cliff_images) //Not cut_overlays() because turf smoothing gets royally fugged.
+		var/list/L = list()
+		for(var/k in cliff_images)
```

Comment:
@xxalpha this is not just a Python thing.

[.NET's `IDictionary<TKey, TValue>` defines a `Values` property too.](https://msdn.microsoft.com/en-us/library/cc673034(v=vs.110).aspx)